src/components/title/index.ts

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Title from "./title";
+
+export default styled(Title)`
+  .text {
+    font-size: 12rem;
+    fill: none;
+    stroke-width: 6;
+    stroke-linejoin: round;
+    stroke-dasharray: 70 330;
+    stroke-dashoffset: 0;
+    -webkit-animation: stroke 6s infinite linear;
+    animation: stroke 6s infinite linear;
+
+    &:nth-child(5n + 1) {
+      stroke: #5c940d;
+      -webkit-animation-delay: -1.2s;
+      animation-delay: -1.2s;
+    }
+    &:nth-child(5n + 2) {
+      stroke: #f5a503;
+      -webkit-animation-delay: -2.4s;
+      animation-delay: -2.4s;
+    }
+    &:nth-child(5n + 3) {
+      stroke: #e9f1df;
+      -webkit-animation-delay: -3.6s;
+      animation-delay: -3.6s;
+    }
+    &:nth-child(5n + 4) {
+      stroke: #56d9cd;
+      -webkit-animation-delay: -4.8s;
+      animation-delay: -4.8s;
+    }
+    &:nth-child(5n + 5) {
+      stroke: #3aa1bf;
+      -webkit-animation-delay: -6s;
```

Comment:
```suggestion
      animation-delay: -6s;
```

I'm not sure about this one, but as long as I know `styled-components` automatically adds vendor prefixes. Give it a try; remove the vendor prefix, run, and inspect. You might see the vendor there.